solana-programs/anchor/audius-data/tests/audius-data.ts

```diff
@@ -0,0 +1,409 @@
+import * as anchor from '@project-serum/anchor';
```

Comment:
I think groccing this test file would be easier if most/all the helpers were moved into a tests/utils.ts file or similar

Reply:
great comment, have done so now!